macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -218,12 +218,35 @@ final class AddressBarButtonsViewController: NSViewController {
         subscribeToButtonsVisibility()
         subscribeToAIChatPreferences()
         setupDaxLogo()
+        setupButtonsCornerRadius()
 
         bookmarkButton.sendAction(on: .leftMouseDown)
         bookmarkButton.normalTintColor = visualStyle.colorsProvider.iconsColor
-        zoomButton.normalTintColor = visualStyle.colorsProvider.iconsColor
         configureAIChatButton()
         privacyEntryPointButton.toolTip = UserText.privacyDashboardTooltip
+        setupButtonPaddings()
+    }
+
+    private func setupButtonPaddings() {
+        if visualStyle.shouldAddPaddingToAddressBarButtons {
```

Comment:
nit pick: you could use guard in place of if here to not have to indent the entire body of the function.